client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala

```diff
@@ -198,11 +198,6 @@ class LifecycleManager(val appUniqueId: String, val conf: CelebornConf) extends
     case StageEnd(shuffleId) =>
       logInfo(s"Received StageEnd request, shuffleId $shuffleId.")
       handleStageEnd(shuffleId)
-    case pb: PbUnregisterShuffle =>
```

Comment:
shall we also update the `proto` file? or add a comment to the `proto` file for future cleanup?

Reply:
`PbUnregisterShuffle` is also used in Master, so I guess we can't delete the message.